greymatter/modules/sidecar/secrets.tf

```diff
@@ -0,0 +1,12 @@
+//resource "aws_ssm_parameter" "access_key" {
```

Comment:
Is this file necessary?

Reply:
no not right now  that was from an attempt at #11  ill delete